sql/5_0_2-to-5_0_3_upgrade.sql

```diff
@@ -431,3 +431,7 @@ INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`, `is_default`
 #IfNotRow2D list_options list_id Sort_Direction option_id 1
 INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`, `is_default`, `activity`) VALUES ('Sort_Direction', '1', 'desc', 20, 0, 1);
 #EndIf
+
+#IfNotColumnType form_eye_mag_prefs ordering smallint(6) NULL
+ALTER TABLE `form_eye_mag_prefs` MODIFY `ordering` smallint(6) NULL;
```

Comment:
missing the default (best to make it exactly same as database.sql):
```
ALTER TABLE `form_eye_mag_prefs` MODIFY `ordering` smallint(6) DEFAULT NULL;
```

Reply:
Done!!